dcp.go

```diff
@@ -2,14 +2,14 @@ package dcp
 
 import (
 	"errors"
+	"github.com/tidwall/sjson"
```

Comment:
Thank you for your PR 🙏🏻 I want to ask a question. Is it possible to get rid of this dependency? I think we can do this masking operation without dependency. Is there any specific reason to choice this one? 😄

Reply:
I tried to do it without library dependency, but json manipulation in golang is really boring.

I find one solution but it doesn't keep fields order. https://stackoverflow.com/a/66493361

sjson is compact, tiny and fast library thats why I decided to use it.
benchmarks: https://github.com/tidwall/sjson/tree/master#performance

Reply:
Hi, 
Maybe we could simplify that process by sending a copy of the config file to the printConfiguration function and then mutating the password as "***" and printing it.  What do you think @ramazan?

Reply:
@emreodabas your idea make sense.  I changed implementation. Thanks